tests/topotests/lib/common_config.py

```diff
@@ -1167,10 +1254,12 @@ def create_debug_log_config(tgen, input_dict, build=False):
                     for daemon, debug_logs in disable_logs.items():
                         for debug_log in debug_logs:
                             debug_config.append("no {}".format(debug_log))
+            if debug_config:
```

Comment:
is this ... idiomatic python? the list does exist - would `len(debug_config) > 0` be clearer?

Reply:
It's definitely idiomatic python :)

Sometimes you even get warnings for using len (not for pure lists, but for things that might be list like e.g., in scapy)